packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -376,15 +376,11 @@ export default function VisualEditor( { styles } ) {
 						{ ! isTemplateMode && (
 							<div
 								className={ classnames(
-									// This wrapper div should have the same
-									// classes as the block list beneath.
-									'is-root-container',
-									'block-editor-block-list__layout',
```

Comment:
Just double-checking, these two lines aren't required because we already have layout styles output that target `edit-post-visual-editor__post-title-wrapper` directly?

Reply:
The post content styles are targeted at those classes [here](https://github.com/WordPress/gutenberg/blob/fix/post-title/packages/edit-post/src/components/visual-editor/index.js#L257), so if we don't remove them they'll still apply to the title. It's also semantically incorrect to use them here, because the title wrapper isn't a block container.

Reply:
Excellent, that all makes sense, thanks for confirming!